docs/tutorials/03_ground_state_solvers.ipynb

```diff
@@ -97,21 +113,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from qiskit.aqua.algorithms import VQE\n",
+    "from qiskit.algorithms import VQE\n",
     "from qiskit.circuit.library import TwoLocal\n",
     "\n",
     "num_qubits = 4\n",
```

Comment:
Can we delete the `num_qubits` line here. It should not be needed as VQE will adjust the number of qubits of teh ansatz when it gets the operator to match it. This was part of design as we did not expect a user to have to know the operator size, especially since we do all sorts of reductions etc. It was brought up in the past - it seems I did not transfer issue from Aqua, since it was about tutorials and could not, but since they are here now I just did #110

Reply:
fixed